lib/modules/specialconfigs/index.js

```diff
@@ -50,9 +70,18 @@ class SpecialConfigs {
 
     this.embark.registerActionForEvent("contracts:deploy:afterAll", (cb) => {
       let afterDeployCmds = self.contractsConfig.afterDeploy || [];
-
       async.mapLimit(afterDeployCmds, 1, (cmd, nextMapCb) => {
-        self.replaceWithAddresses(cmd, nextMapCb);
+        async.waterfall([
+          function replaceWithAddresses(next) {
+            self.replaceWithAddresses(cmd, next);
+          },
+          function replaceWithENSAddress(result, next) {
+            if(self.embark.config.namesystemConfig && !self.embark.config.namesystemConfig.enabled) {
```

Comment:
I just figured, instead of checking if ENS is activated here, you can add the check in the ENS module, make the event available even if ENS is deactivated and return an error if it is when we call the event.

Reply:
Hi I added a event to ens module and also using that event.